Source/Core/DolphinWX/FifoPlayerDlg.cpp

```diff
@@ -80,197 +80,197 @@ void FifoPlayerDlg::CreateGUIControls()
 	m_Notebook = new wxNotebook(this, wxID_ANY);
 
 	{
-	m_PlayPage = new wxPanel(m_Notebook, wxID_ANY, wxDefaultPosition, wxDefaultSize, wxTAB_TRAVERSAL);
-	wxBoxSizer* sPlayPage;
-	sPlayPage = new wxBoxSizer(wxVERTICAL);
+		m_PlayPage = new wxPanel(m_Notebook, wxID_ANY, wxDefaultPosition, wxDefaultSize, wxTAB_TRAVERSAL);
```

Comment:
I would indent anyway. If it's a block, no reason why it should be at the same level as the previous block.